server/weaverbird/steps/totals.py

```diff
@@ -19,7 +26,7 @@ class Config(PopulatedWithFieldnames):
 class TotalsStep(BaseStep):
     name = Field('totals', const=True)
     total_dimensions: List[TotalDimension] = Field(alias='totalDimensions')
-    aggregations: List[Aggregation] = Field(min_items=1)
+    aggregations: Sequence[Aggregation]
```

Comment:
@fspot  I needed to replace List by Sequence because of a weird error about covariance. However, we cannot have a min_items constraint with a Sequence. I could add it as a root_validator, is it an acceptable workaround ?